Bank/InvestAccount.cs

```diff
@@ -0,0 +1,44 @@
+class InvestAccount
+{
+    private static readonly Random random = new Random();
+    private static readonly int maxDelta = 2;
+    private static readonly int minDelta = -1;
+    
+    public int interest{get; set;}
+    public double money{get; set;}
+
+    public int GetInterest(){return this.interest;}
+    public double GetMoney(){return this.money;}
+
+
+    public InvestAccount()
+    {
+        this.money = 0;
+        this.interest = 3;
+    }
+    public InvestAccount(double initDeposit)
+    {
+        this.money = initDeposit;
+        this.interest = 3;
+    }
+
+    public void YearlyUpdate()
```

Comment:
Method's name should indicate what it does mostly as a verb. E.g. UpdateYear or IncreaseYear.